scripts/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+ninja -C build
```

Comment:
this script has no purpose and will confuse people in it's current state

Reply:
It's purpose is to mirror `build.ps1`

```json
    {
      "type": "process",
      "label": "Build Bun",
      "dependsOn": ["Setup Environment"],
      "command": "scripts/build.sh",
      "windows": {
        "command": "scripts/build.ps1"
      },
    }
```

Reply:
Going to remove for now though